#List of event names for which there is a scheduled time
string[] event_names

#Corresponding list of scheduled times for the events
float32[] event_schedules
